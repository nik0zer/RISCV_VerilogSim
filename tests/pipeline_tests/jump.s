.section .text
.global _start

_start:
    addi x1, x0, 1
loop:
    addi x1, x1, 1
    jal x4, loop
    nop
    nop
    nop
    nop
    nop
    nop
    nop
    nop
    nop
    nop
    nop
    nop
    nop
    nop
    nop