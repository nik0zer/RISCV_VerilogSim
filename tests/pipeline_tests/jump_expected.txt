x
x
x
x
0000000000000001
0000000000000000
0000000000000000
0000000000000000
0000000000000002
0000000000010018
0000000000000000
0000000000000000
0000000000000000
0000000000000003
0000000000010018
0000000000000000
0000000000000000
0000000000000000
0000000000000004
0000000000010018
0000000000000000
0000000000000000
0000000000000000
